src/components/footer/bottom-bar.tsx

```diff
@@ -0,0 +1,22 @@
+import styles from "./footer.module.scss";
+
+const BottomBar = () => {
+  return (
+    <div className={styles.bottomBar}>
+      <div className={styles.bottomBar_divider}></div>
+      <div className={styles.bottomBar_content}>
+        <div className={styles.bottomBar_leftText}>
+          {"Copyright © 2021 UI8 LLC. All rights reserved"}
+        </div>
+        <div className={styles.bottomBar_rightFrame}>
```

Comment:
frame would be bad naming here. Iframe in web development refers to a specific object: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe